src/index.tsx

```diff
@@ -167,6 +174,13 @@ const ReactInputVerificationCode = ({
     if (pValue !== value.join('')) setValue(pValue.split(''));
   }, [pValue]);
 
+
+  const renderItemValue = (value: string) => {
+    if (!value) return placeholder;
+
+    return type === InputType.PASSWORD ? '●' : value;
+  }
```

Comment:
This won't work because `value` is also defined earlier, leading to an always true return
I also used a small dot 🙂 

```suggestion
  const passwordMask = '•' // this could even go into a dedicated prop, not mandatory

  const renderItemText = (itemValue: string) => {
    if (itemValue === placeholder) return placeholder;
    return type === 'password' ? passwordMask : itemValue;
  };
```